frontend/src/App.test.tsx

```diff
@@ -6,10 +6,15 @@ describe("App component", () => {
   it("renders the heading with correct text", () => {
     render(<App />);
 
-    // Find the heading element
-    const headingElement = screen.getByRole("heading", { level: 1 });
+    // Get all headings
+    const headingElements = screen.getAllByRole("heading", { level: 1 });
```

Comment:
There are now multiple `h1` headings rendered in the main page of the SPA, so need to account for that